easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.1-2023a.yml

```
@@ -17,3 +17,6 @@ easyconfigs:
         from-pr: 20595
   - GATK-4.5.0.0-GCCcore-12.3.0-Java-17.eb
   - WhatsHap-2.2-foss-2023a.eb
+  - BLAST+-2.14.1-gompi-2023a.eb:
+      options:     
+        from-pr: 20751
```

Could you try to use PR 20784 and build this for one Arm target, just to verify that this solves the issue?
```suggestion
        from-pr: 20784
```